copter/source/docs/precision-landing-with-irlock.rst

```diff
@@ -43,7 +43,7 @@ Parameters
 - :ref:`PLND_RET_BEHAVE<PLND_RET_BEHAVE>` how the moves to last valid landing target report on a retry
 - :ref:`PLND_ALT_MIN<PLND_ALT_MIN>` Vehicle will continue landing vertically even if target is lost below this height. This needs a rangefinder to work. Set to zero to disable this.
 - :ref:`PLND_ALT_MAX<PLND_ALT_MAX>` Vehicle will continue landing vertically until this height if target is not found. Below this height if landing target is not found, landing retry/failsafe might be attempted. This needs a rangefinder to work. Set to zero to disable this.
-- :ref:`PLND_OPTIONS<PLND_OPTIONS>` set bit 0 if landing target is moving, set bit 1 to allow the precision landing to continue after a manual reposition.
+- :ref:`PLND_OPTIONS<PLND_OPTIONS>` set bit 0 if landing target is moving, set bit 1 to allow the precision landing to continue after a manual reposition, and set bit 2 to override additional final landing speed slow downs for precise positioning.
```

Comment:
I think it maybe clearer if we add "Setting the second bit makes the landing descent profile similar to the normal land mode (i.e land mode without precision landing enabled)"